sct.py

```diff
@@ -505,38 +504,6 @@ def list_ami_versions(region: str, arch: AwsArchType):
     click.echo(tbl.get_string(title="Scylla AMI versions"))
 
 
-@cli.command('list-ami-branch', help="""list Amazon Scylla branched AMI versions
-    \n\n[VERSION] is a branch version to look for, ex. 'branch-2019.1:latest', 'branch-3.1:all'""")
-@click.option('-r', '--region',
-              type=CloudRegion(cloud_provider="aws"),
-              default='eu-west-1',
-              help="a region to look for AMIs (default: eu-west-1)")
-@click.option('-a', '--arch',
-              type=click.Choice(AwsArchType.__args__),
-              default='x86_64',
-              help="architecture of the AMI (default: x86_64)")
-@click.argument('version', type=str, default='branch-3.1:all')
-def list_ami_branch(region: str, arch: AwsArchType, version: str):
-    add_file_logger()
-
-    def get_tags(ami):
-        return {i['Key']: i['Value'] for i in ami.tags}
-
-    if ":" not in version:
-        version += ":all"
-
-    tbl = PrettyTable(field_names=["Name", "ImageId", "CreationDate", "BuildId", "Test Status"], align="l")
```

Comment:
Seem like you are throwing away the "Test Status".

it a feature we have that can report a longevity result back to the used AMI, as a tag.

we didn't used it, and if yanking it, maybe we should yank all of it.

Reply:
Yup, I removed it as it seems we don't use these values. What do you mean by yanking "all of it"?

Reply:
We have `tag_ami_with_result` in `tester.py`, but it is getting the result from the event summary. From what I can tell there is no connection between the code removed here and the actual tagging method.